### PR TITLE
TRON-26:Sell Car Page Design

### PR DESCRIPTION
**Pull request by Hassan0112**
Ticket:https://pakw.atlassian.net/browse/TRON-26
**Description**
**What?**
Make a design for sell car page.
**Acceptance** Criteria:

- Continue design changes for sell car page the next midway of the page is attached you can take look at it. Use appropriate font family, text-size, color, font-aweseom etc…

- You can inspect the page and look at the respective font-size, font-family, text-color, font-awesome, images etc.

- Design should be responsive and should match exactly with the given template.

### CODE REVIEW

File: app/views/used_cars/post_ad.html.erb [hunk 1]
```diff
@@ -52,4 +52,61 @@
...
...06 19.8633 31.9162 20.0029 32.6102 20.2822C33.3042 20.5615 33.8925 20.9678 34.3749 21.501C34.8658 22.0342 35.2382 22.6859 35.4921 23.4561C35.746 24.2262 35.8729 25.098 35.8729 26.0713C35.8729 26.8923 35.746 27.6751 35.4921 28.4199C35.2382 29.1562 34.8531 29.8164 34.3368 30.4004C33.8205 30.9759 33.1731 31.4287 32.3944 31.7588C31.6158 32.0889 30.6932 32.2539 29.6268 32.2539C28.8313 32.2539 28.0611 32.1354 27.3163 31.8984C26.58 31.6615 25.9156 31.3102 25.3231 30.8447C24.7391 30.3792 24.2694 29.8164 23.9139 29.1562C23.5669 28.4876 23.385 27.7259 23.368 26.8711H26.9989C27.0497 27.3958 27.1851 27.8486 27.4052 28.2295C27.6337 28.6019 27.9341 28.8896 28.3065 29.0928C28.6789 29.2959 29.1148 29.3975 29.6141 29.3975C30.0796 29.3975 30.4774 29.3086 30.8075 29.1309C31.1376 28.9531 31.4042 28.7077 31.6073 28.3945C31.8104 28.0729 31.9585 27.7005 32.0516 27.2773C32.1532 26.8457 32.204 26.3802 32.204 25.8809C32.204 25.3815 32.1447 24.9287 32.0262 24.5225C31.9078 24.1162 31.7258 23.765 31.4803 23.4688C31.2349 23.1725 30.9218 22.944 30.5409 22.7832C30.1685 22.6224 29.7326 22.542 29.2333 22.542C28.5562 22.542 28.0315 22.6478 27.6591 22.8594C27.2951 23.071 26.9904 23.3122 26.745 23.583Z" fill="black"/>
```
Comment (Adil449) [inline app/views/used_cars/post_ad.html.erb:76]: Remove this and use jpg instead of svg.

File: app/views/used_cars/post_ad.html.erb [hunk 2]
```diff
@@ -52,4 +52,61 @@
...
...23C21.6161 30.6309 22.2551 30.9609 22.7714 31.3926C23.2876 31.8158 23.677 32.3193 23.9393 32.9033C24.2017 33.4788 24.3329 34.1178 24.3329 34.8203C24.3329 35.6836 24.1678 36.4538 23.8378 37.1309C23.5077 37.7995 23.0422 38.3665 22.4413 38.832C21.8488 39.2975 21.1548 39.653 20.3593 39.8984C19.5637 40.1354 18.6962 40.2539 17.7567 40.2539C16.9781 40.2539 16.2121 40.1481 15.4589 39.9365C14.7141 39.7165 14.037 39.3906 13.4276 38.959C12.8267 38.5189 12.3443 37.9688 11.9803 37.3086C11.6249 36.64 11.4471 35.8486 11.4471 34.9346H15.1161C15.1161 35.4085 15.2346 35.8317 15.4716 36.2041C15.7085 36.5765 16.0344 36.8685 16.4491 37.0801C16.8723 37.2917 17.342 37.3975 17.8583 37.3975C18.4423 37.3975 18.9416 37.2917 19.3563 37.0801C19.7795 36.86 20.1011 36.5553 20.3212 36.166C20.5497 35.7682 20.6639 35.307 20.6639 34.7822C20.6639 34.1051 20.5412 33.5635 20.2958 33.1572C20.0503 32.7425 19.6991 32.4378 19.2421 32.2432C18.785 32.0485 18.2434 31.9512 17.6171 31.9512H15.662ZM28.9667 21.5156H32.1913L36.9393 35.0869L41.6874 21.5156H44.912L38.2343 40H35.6444L28.9667 21.5156ZM27.2274 21.5156H30.4393L31.0233 34.7441V40H27.2274V21.5156ZM43.4393 21.5156H46.6639V40H42.8553V34.7441L43.4393 21.5156Z" fill="black"/>
```
Comment (Adil449) [inline app/views/used_cars/post_ad.html.erb:89]: Remove this and use jpg instead of svg.

File: app/views/used_cars/post_ad.html.erb [hunk 3]
```diff
@@ -52,4 +52,61 @@
...
...712 46.651 41.0505 46.4824 40.9639C46.3184 40.8727 46.1042 40.8271 45.8398 40.8271C45.6211 40.8271 45.432 40.8659 45.2725 40.9434C45.113 41.0163 44.9899 41.1165 44.9033 41.2441C44.8213 41.3672 44.7803 41.5039 44.7803 41.6543C44.7803 41.7682 44.8031 41.8708 44.8486 41.9619C44.8988 42.0485 44.9785 42.1283 45.0879 42.2012C45.1973 42.2741 45.3385 42.3424 45.5117 42.4062C45.6895 42.4655 45.9082 42.5202 46.168 42.5703C46.7012 42.6797 47.1774 42.8232 47.5967 43.001C48.016 43.1742 48.3486 43.4111 48.5947 43.7119C48.8408 44.0081 48.9639 44.3978 48.9639 44.8809C48.9639 45.209 48.891 45.5098 48.7451 45.7832C48.5993 46.0566 48.3896 46.2959 48.1162 46.501C47.8428 46.7015 47.5146 46.8587 47.1318 46.9727C46.7536 47.082 46.3275 47.1367 45.8535 47.1367C45.1654 47.1367 44.582 47.0137 44.1035 46.7676C43.6296 46.5215 43.2695 46.2093 43.0234 45.8311C42.7819 45.4482 42.6611 45.0563 42.6611 44.6553H44.5273C44.5365 44.9242 44.6048 45.1406 44.7324 45.3047C44.8646 45.4688 45.0309 45.5872 45.2314 45.6602C45.4365 45.7331 45.6576 45.7695 45.8945 45.7695C46.1497 45.7695 46.3617 45.7354 46.5303 45.667C46.6989 45.5941 46.8265 45.4984 46.9131 45.3799C47.0042 45.2568 47.0498 45.1156 47.0498 44.9561Z" fill="black"/>
```
Comment (Adil449) [inline app/views/used_cars/post_ad.html.erb:101]: Remove this and use jpg instead of svg.

File: app/views/used_cars/post_ad.html.erb (hunk 1 above)
Comment (Hassan0112) [reply app/views/used_cars/post_ad.html.erb]: done

File: app/views/used_cars/post_ad.html.erb (hunk 2 above)
Comment (Hassan0112) [reply app/views/used_cars/post_ad.html.erb]: done

File: app/views/used_cars/post_ad.html.erb (hunk 3 above)
Comment (Hassan0112) [reply app/views/used_cars/post_ad.html.erb]: done

**Review by Adil449 — APPROVED** [review]

**Pull request merged**